apptainer/moose-dev.def

```diff
@@ -152,9 +164,42 @@ From: {{ APPTAINER_FROM }}
     # This is redundant; hopefully we can use the one from conda in the future
     WASP_SRC_DIR=${WASP_SRC_DIR} ${WASP_BUILD_SCRIPT} -D CMAKE_INSTALL_PREFIX:STRING=${WASP_DIR}
 
-    # Install node.js
+    # Install node.js. Requested by dschwen for bison
     dnf install -y nodejs npm
 
+{% if PROFILING is defined %}
+    # Install gperftools
+    GPERFTOOLS_VERSION=gperftools-2.15
+    cd ${ROOT_BUILD_DIR}
+    git clone -b ${GPERFTOOLS_VERSION} --single-branch https://github.com/gperftools/gperftools.git
+    GPERFTOOLS_SRC=${ROOT_BUILD_DIR}/gperftools
+    cd ${GPERFTOOLS_SRC}
+    ./autogen.sh
+    ./configure --prefix={{ GPERF_DIR }} --enable-frame-pointers
```

Comment:
Are people hoping to do a lot of heap profiling? That would be the reason I'd recommend going to the frame-pointer route. Otherwise using libunwind gives more accurate stack traces, particularly in my experience if fortran libraries like mumps are involved

Reply:
No one seems to be commenting on this. And I am only adding what other have been asking for.

Reply:
We talked about this. This is fine as is, and we can change it around later if needed.